lib/dissoc/dissoc.performance.js

```diff
@@ -0,0 +1,17 @@
+const generateObjects = require('../_internal/helpers/performance').generateObjects
+
+module.exports = function() {
+  const [small, medium, big] = generateObjects([1e3, 1e4, 1e5])
+  const getProp = (obj, length) => Object.keys(obj)[length / 2]
```

Comment:
`Object.keys(obj).length` has a length you specify below, maybe use it instead?

Reply:
I do not use length of `Object.keys(obj)`, I use length from arguments. I will use that length from constants, as you said

Reply:
I mean the length of arrays will be the same, doesn't matter where we get it

Reply:
It's not the same